pyne/utils.py

```diff
@@ -11,72 +11,6 @@
 class QAWarning(UserWarning):
     pass
 
-time_conv_dict = {'as': 1e-18,
```

Comment:
I think that this dict (but with out the energy keys) and the original `to_sec()` fucntion should be restored.  There is no reason to break API here, and  I think that this function may be used by some down stream packages.